src/deprecation.cpp

```diff
@@ -12,6 +12,16 @@
 static const std::string CLIENT_VERSION_STR = FormatVersion(CLIENT_VERSION);
 
 void EnforceNodeDeprecation(int nHeight, bool forceLogging) {
+
+    // Do not enforce deprecation in regtest or on testnet
+    const CChainParams& chainparams = Params();
+    std::string networkID = chainparams.NetworkIDString();
```

Comment:
Can just be ``std::string networkID = Params().NetworkIDString();``. (Non-blocking comment.)